src/ArrayType.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+/**
+ * @package Generator
+ */
+namespace Wsdl2PhpGenerator;
+
+use \Exception;
+use Wsdl2PhpGenerator\PhpSource\PhpDocComment;
+use Wsdl2PhpGenerator\PhpSource\PhpDocElementFactory;
+use Wsdl2PhpGenerator\PhpSource\PhpFunction;
+
+/**
+ * ArrayType
+ *
+ * @package Wsdl2PhpGenerator
+ * @author Fredrik Wallgren <fredrik.wallgren@gmail.com>
+ * @license http://www.opensource.org/licenses/mit-license.php MIT License
```

Comment:
Please remove the `@author` and `@license` annotations.

This has a been community project for a while even though there a similar traces of this throughout existing classes. The license information is included in the `LICENSE` file.